### PR TITLE
*/*: prefix homepage update

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

another few homepage and `http` -> `https` fixes for prefix packages

### CODE REVIEW

File: app-portage/prefix-toolkit/prefix-toolkit-10-r1.ebuild [hunk 1]
```diff
@@ -4,7 +4,7 @@
 EAPI=7
 
 DESCRIPTION="Utilities for users of Gentoo Prefix"
-HOMEPAGE="https://prefix.gentoo.org/"
+HOMEPAGE="https://wiki.gentoo.org/wiki/Project:Prefix"
```
Comment (grobian) [inline app-portage/prefix-toolkit/prefix-toolkit-10-r1.ebuild:7]: what's wrong with the prefix.gentoo.org url here?

Comment (mm1ke) [reply app-portage/prefix-toolkit/prefix-toolkit-10-r1.ebuild:7]: there's nothing particularly wrong with prefix.gentoo.org.
I made the change to save an redirect to to `https://wiki.gentoo.org/wiki/Project:Prefix`
It's only a minor change, but i think it a nice to have change.

**Comment by mm1ke** [comment]
@grobian 
Ping, any chance we can merge this?
If you're not fine the gentoo Homepage changes i'll remove them and only keep the `http` -> `https` + LICENSE fixes.

**Comment by grobian** [comment]
I'd prefer to keep the alias, but feel free to merge.

**Comment by mm1ke** [comment]
> I'd prefer to keep the alias, but feel free to merge.

No problem, i've removed the commits. BTW, i can't merge, i'm not a dev ;)

**Comment by grobian** [comment]
ah, is someone working with you, or do you need a hand?

**Comment by ConiKost** [comment]
I guess me ;-) Merged the rest non prefix stuff, since it makes sense to update it.

**Comment by mm1ke** [comment]
> I guess me ;-) Merged the rest non prefix stuff, since it makes sense to update it.

Thanks @ConiKost :-)

**Pull request closed without merging**